src/components/EnterpriseList/index.jsx

```diff
@@ -182,4 +182,4 @@ EnterpriseList.propTypes = {
   error: PropTypes.instanceOf(Error),
 };
 
-export default EnterpriseList;
+export default withRouter(EnterpriseList);
```

Comment:
`CLARIFICATION`: so we now return a withRouter surrounded component instead of the regular component, okay, I believe this means it's an in place replacement for wherever this component is used, just that it has access to extra props right?

Reply:
This is correct. This is just passing route props into the component, but doesn't change anything else.